docs/guides/controller-autoconfiguration.md

```diff
@@ -38,32 +39,40 @@ Having automatically configured joypads makes it a lot easier to navigate the Re
 
 ## Installing or updating joypad profiles
 
-![downloading joypad profiles](../image/retroarch/input/update-joypads.jpg)
+![image](https://github.com/davidhedlund/docs/assets/6416073/e2c11f35-fef1-402b-a975-e092b05d4dcc)
```

Comment:
It should be located here. docs/image/guides

Reply:
Thank you for reaching out. I'm just overloaded. It's ok if we skip the image, unless someone else wants to solve this. It's no biggie for me, the important thing is to merge since I made a lot of updates.